experimental/dds/tree/src/LogViewer.ts

```diff
@@ -320,9 +326,14 @@ export class CachingLogViewer implements LogViewer {
 		}
 	}
 
-	public async getEditResult(revision: Revision): Promise<EditCacheEntry> {
+    public async tryGetEditResult(revision: Revision): Promise<EditCacheEntry | undefined> {
 		const startingPoint = this.getStartingPoint(revision);
```

Comment:
@robertobe-ms this "starting point" is the place that we need to guarantee works for the available edits/revisions.